src/Rules/Rules.cs

```diff
@@ -12,7 +12,9 @@ namespace Axe.Windows.Rules
     /// <summary>
     /// Use this class to run rules and get information about any rule in the Rules assembly.
     /// </summary>
+    #pragma warning disable CA1724
```

Comment:
Inconsistent indents
```suggestion
#pragma warning disable CA1724
```